delivery_roulier/models/stock_move_line.py

```diff
@@ -39,3 +39,18 @@ def get_unit_price_for_customs(self):
     def get_sale_order_line(self):
         self.ensure_one()
         return self.move_id.sale_line_id
+
+    def write(self, vals):
```

Comment:
My big question: Why is this specific to roulier ??

Reply:
It's not specific to roulier.
Somehow after `stock.picking`::`_put_in_pack` call is finished `quant_id` is not assigned to a `stock.quant.package`.
But, from what I understand during tests debug `quant_id` is expected to be assigned.
So, I suppose there is a refactor of delivery between v16 and v17, which should be handled.
Any better ideas?
Thanks for review